charts/furiosa-metrics-exporter/templates/daemonset.yaml

```diff
@@ -21,13 +21,20 @@ spec:
       priorityClassName:  {{ .Values.daemonSet.priorityClassName | default "system-node-critical" }}
       containers:
         - name: {{ include "furiosa-metrics-exporter.name" . }}
-          image: {{ .Values.daemonSet.image.repository | default "docker.io/furiosaai/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
-          command: ["./main"]
+          image: {{ .Values.daemonSet.image.repository | default "registry.corp.furiosa.ai/furiosa/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
```

Comment:
don't try to fix defaulting :)

Reply:
Oh, It just copied from [this repo](https://github.com/furiosa-ai/furiosa-metrics-exporter) by mistake.

Fixed at 45932130